### PR TITLE
Update Spanish translation for version 2.0

### LINKED ISSUE #36: Update Spanish translation for version 2.0
Version `1.2` of **vrms-rpm** had a Spanish translation. Given the major overhaul of the program for version `2.0`, this translation can no longer be used. It'd be awesome if we could update the translation (or create an entirely new one) for the new version.

Files containing the old translation:
- `lang/outdated/es.po`
- `man/outdated/es.man`

New text (in English) to be translated:
- `lang/en.po`
- `man/en.man`

### PR DESCRIPTION
**Pull request by mroutis**
Close #36

### CODE REVIEW

File: lang/es.po [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+msgstr "Uso: vrms-rpm [opciones]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Muestra arte ASCII de rms cuando no se encuentra ningún paquete privado,\n"
+       "    o cuando los paquetes privados son un 10%% o más del total.\n"
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Controla si la terminal debería de usar sequencias de escape\n"
+       "    para imprimir colores. El predefinido es 'auto', que utiliza colores\n"
+       "    cuando se imprime en la terminal, sólo si no se redirecciona a un archivo o a un pipe.\n"
```
Comment (suve) [inline lang/es.po]: This message should not go over 80 characters per line. You can reword it to be shorter, or just add another linebreak (`\n`).

File: lang/es.po [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+msgid "HELP_OPTION_DESCRIBE\n"
+msgstr "    Cuando enlistas paquetes, incluye el resumen de los paquetes (descripción corta).\n"
```
Comment (suve) [inline lang/es.po]: Same as above.

File: lang/es.po [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+msgid "HELP_OPTION_EXPLAIN\n"
+msgstr "    Cuando enlistas paquetes, muestra sus licencias\n"
+       "    para justificar la clasificación de libre o privado.\n"
+
+msgid "HELP_OPTION_HELP\n"
+msgstr "    Muestra esta ayuda y termina el programa.\n"
+
+msgid "HELP_OPTION_IMAGE\n"
+msgstr "    Similar a la opción --ascii, pero muestra una imagen usando bloques (caracteres) Unicode\n"
```
Comment (suve) [inline lang/es.po]: Same as above.

File: lang/es.po [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+       "    y códigos de escape para terminales que soportan 256 colores.\n"
+
+msgid "HELP_OPTION_LICENCELIST\n"
+msgstr "    Especifica la lista de buenas licencias para usar. ARCHIVO puede ser una ruta\n"
```
Comment (suve) [inline lang/es.po]: Same as above.

File: lang/es.po [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+msgstr ""
+"Language: es\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "es translation by Mr. Outis\n"
```
Comment (suve) [inline lang/es.po]: I suggest putting a sentence in Spanish here, instead of English.

File: man/es.man [hunk 6]
```diff
@@ -0,0 +1,96 @@
+.TH VRMS-RPM 1 "2018-05-18"
```
Comment (suve) [inline man/es.man]: This date should reflect the day you translated the file.

**Comment by suve** [comment]
@mroutis are you planning on updating this PR?

**Comment by mroutis** [comment]
sorry, @suve, updating it right now!

**Comment by mroutis** [comment]
Hope that we are all set, @suve, feel free to ping me if any update is needed.

**Comment by suve** [comment]
Thanks! Merged.
